Jenkinsfile_promote_salt_packages

```diff
@@ -9,49 +9,83 @@ properties([
-                sh "curl https://build.opensuse.org/package/view_file/systemsmanagement:saltstack:products:testing:debian/salt/_service?expand=0 | grep salt_${salt_version}.orig.tar.g"
-            }
-        }
+                    echo "Check that 'products:testing' and 'products:testing:debian' are not set to MU branches"
+                    sh "! curl https://build.opensuse.org/package/view_file/systemsmanagement:saltstack:products:testing/salt/_service?expand=1 | grep MU/${mu_version}"
+                    sh "! curl https://build.opensuse.org/package/view_file/systemsmanagement:saltstack:products:testing:debian/salt/_service?expand=1 | grep MU/${mu_version}"
 
-        stage('Create temporary OBS environment') {
-            steps {
-                sh "mkdir /tmp/salt-promote-pipeline-env || true"
+                    echo "Check the source tarball is properly named to salt_${salt_version}.orig.tar.gz in 'products:testing:debian'"
+                    sh "curl https://build.opensuse.org/package/view_file/systemsmanagement:saltstack:products:testing:debian/salt/_service?expand=0 | grep salt_${salt_version}.orig.tar.gz"
+                }
             }
         }
 
         stage('Promote Salt testing RPM packages') {
             steps {
                 echo 'Promote Salt testing packages from "products:testing" to "products"'
-                sh "osc copypac systemsmanagement:saltstack:products:testing salt systemsmanagement:saltstack:products"
-                sh "osc copypac systemsmanagement:saltstack:products:testing python-contextvars systemsmanagement:saltstack:products"
-                sh "osc copypac systemsmanagement:saltstack:products:testing python3-contextvars systemsmanagement:saltstack:products"
-                sh "osc copypac systemsmanagement:saltstack:products:testing python-immutables systemsmanagement:saltstack:products"
-                sh "osc copypac systemsmanagement:saltstack:products:testing python3-immutables systemsmanagement:saltstack:products"
-                sh "osc copypac systemsmanagement:saltstack:products:testing py26-compat-salt systemsmanagement:saltstack:products"
-                sh "osc copypac systemsmanagement:saltstack:products:testing py26-compat-tornado systemsmanagement:saltstack:products"
-                sh "osc copypac systemsmanagement:saltstack:products:testing py26-compat-msgpack-python systemsmanagement:saltstack:products"
-                sh "osc copypac systemsmanagement:saltstack:products:testing py27-compat-salt systemsmanagement:saltstack:products"
-                sh "osc copypac systemsmanagement:saltstack:products:old:testing salt systemsmanagement:saltstack:products:old"
+                sh "python3 promote_packages.py -s systemsmanagement:saltstack:products:testing -t systemsmanagement:saltstack:products packages"
+                sh "python3 promote_packages.py -s systemsmanagement:saltstack:products:old:testing -t systemsmanagement:saltstack:products:old packages"
```

Comment:
Hmmm, good catch. Salt 3000 is EOL, we do not maintain them anymore.

I'm removing this part.

Reply:
Those are not for Salt 3000, `products:old` is for SLE 11 / RHEL 6.

Reply:
Oops, that is correct. I think it makes also sense to remove it from here. In case an update for Salt 2016.11.10 is really required, we will do "copypac" manually.